packages/synapse-react-client/src/mocks/entity/mockReleaseCardsTable.ts

```diff
@@ -0,0 +1,59 @@
+import { EntityHeader, TableEntity } from '@sage-bionetworks/synapse-types'
+import { MOCK_USER_ID } from '../user/mock_user_profile'
+import { MockEntityData } from './MockEntityData'
+import mockProject from './mockProject'
+
+export const MOCK_RELEASE_CARDS_TABLE_ID = 'syn40001'
+const MOCK_RELEASE_CARDS_TABLE_NAME = 'releases table'
+const parentId = mockProject.id
+
+export const mockReleaseCardsTable = {
```

Comment:
Create a mock table to drive the release cards in Storybook until a public entity is created for the GENIE portal.